### ISSUE TITLE
Where are the Bash communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/bash#166]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is Bash supported by one or more large organizations?
* Does Bash have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about Bash? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for Bash? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Bash? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in Bash?
* Are there popular newsletters for Bash?
* Is Bash taught at programming bootcamps? (If so, what are some examples?)
* Is Bash taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about Bash?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by hgvanpariya** [comment]
Shell script is mainly used by semiconductor industries to process Verilog, System Verilog, VHDL file.

**Comment by kytrinyx** [comment]
Thanks, @hgvanpariya. My impression is that there's no particular communities focused on Bash, but that rather it's used in a lot of places, and there are maybe some smaller, disparate groups of people who use it and care about it.

**Comment by roger-fang** [comment]
Mainly developers using Linux and MacOS (UNIX) use Bash (by default). Microsoft now includes Bash in Windows 10 Anniversary Update.

**Comment by budmc29** [comment]
I don't know if there are bash communities, but most of the times people discuss about bash in groups made for that purpose, maybe something like "Unix one liners", or places like code golf.

Usually bash enthusiasts are Gnu Linux / Mac enthusiasts that love to work with the CLI and it's tools.

**Comment by guygastineau** [comment]
This is a great issue.  I think a lot of the bash and sh community is tied (rather loosely) through forums like stack exchange where Linux users often search for the commands to do what they want in bash.

I will try to post on some of those forums to direct people to exercise for advancing their bash skills.

It also seems like @NobbZ might know a good number of bashers and *nix she'll enthusiasts.

Furthermore, I will try to get fellow arch Linux users interested in exercise.io in general with the expectation that they can't resist doing the bash track.

Possible problem: 
How far out are we from bash being up on V2?  If I send people to exercism.io and bash isn't up, then they might not check again?

**Comment by NobbZ** [comment]
As I have been summoned ;)

I do know a lot of people that use *nix on a daily base, for personal use as well as for business, but about no-one actually scripts in bash.

All prefer to use either python or ruby for one-off-scripts.

Also all of them are using zsh as a daily driver as the available tab-completion scripts are better than their bash-versions most of the time.

And, a very big pitty is that no one cares for exercism. I can't convince anyone to take a look :(

**Comment by guygastineau** [comment]
@NobbZ
Thank you for providing this information. 
It is useful to know that some people Don't want to join exercism and that many *nix users don't write bash scripts anymore.

**Comment by NobbZ** [comment]
SEs superuser site and similar do prove that there are actually enough users still doing plain bash scripting.

But people I know, do not use it for scripting.

Of course, sometimes we write bash scripts as well, but those are limited to very basic tasks, as installing the agent of our config management tool and setting up hostname and other facts that the agent needs to identify the role of the machine in question.

Those scripts are often without much logic.

I am sure, there are purists out there that do a lot more bashscripting than me and my friends, but I haven't met them yet.

**Comment by guygastineau** [comment]
### UPDATE: the bash track is now active on exercism.io.
- I will be attempting to direct bash users to the site.
- Maybe our track can become the hub for a healthy community or bash users!

**Comment by hgvanpariya** [comment]
@guygastineau : it seems some bug, when I click on the `bash` it is throwing error.

**Comment by hgvanpariya** [comment]
<img width="1239" alt="screen shot 2018-07-19 at 11 04 14 pm" src="https://user-images.githubusercontent.com/1458819/42959989-3a266936-8ba8-11e8-83c8-c98d5fe9148c.png">

**Comment by guygastineau** [comment]
@hgvanpariya interesting. It isn't doing that for me, but is does happen if I click on the mentor but.

- have you already joined the track?
- can you access it by entering `exercism.io/my/tracks/bash` in the URL?

**Comment by hgvanpariya** [comment]
@guygastineau : while accessing this URL (https://exercism.io/tracks/bash) it shows the error page 
I also tried in Incognito mode also.
(Others can also check, if they are not getting error then it might be specific to me)

**Comment by guygastineau** [comment]
@hgvanpariya that is unfortunate. Would you mind making a new issue on exercism/exercism.io?

I'm afraid this isn't the right place to continue discussing this problem.

**Comment by budmc29** [comment]
@kytrinyx do we need to do anything else about this issue?

**Comment by kytrinyx** [comment]
No, I think we have what we need, thanks!
